cases/case_catalog.md

```diff
@@ -10,5 +9,127 @@ Is this real data or dummy data? How many employees? And special constraints?
 - number of employees: 371
 - ...
 
-## Case 3:
-Within this case the automatically extracted data from the database gets stored...
+## Case 3: - Real Data
+Within this case the automatically extracted data from the database is stored. This excludes the `wishes_and_blocked.json` file as this includes all the manually submitted wishes of employees as well as special circumstances such as health or family-related restrictions. All different files are listed below with the corresponding description of each field within them.
```

Comment:
```suggestion
In this case, the automatically extracted data from the database is stored. The `wishes_and_blocked.json` file is an exception because it includes all the manually submitted wishes of employees, as well as special circumstances, such as health- or family-related restrictions. The different files and their corresponding descriptions are listed below.
```